### ISSUE TITLE
ms_tabNavigationEnabled not working

### ISSUE THREAD

**Issue by nubsoft** [stepstone-tech/android-material-stepper#105]
Hi,  `ms_tabNavigationEnabled=false` not working , When I'm on tab number 2 , I still can click on tab number 1 and goes to that tab directly, which shouldn't be possible when `ms_tabNavigationEnabled=false`

```
<com.stepstone.stepper.StepperLayout
    android:id="@+id/stepperLayout"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:orientation="vertical"
    app:ms_stepperType="tabs"
    app:ms_stepperFeedbackType="tabs|content|disabled_bottom_navigation"
    app:ms_tabNavigationEnabled="false"
    app:ms_backButtonText="Back"
    app:ms_nextButtonText="Next"
    app:ms_completeButtonText="END"/>```

**Comment by zawadz88** [comment]
Hi @nubsoft,
Thanks for reporting this! From what I can see it's an issue when `ms_tabNavigationEnabled` is used together with `tabs` stepper feedback and the issue is in `TabsStepperFeedbackType#hideProgress` where tab navigation is always enabled.
It should check first if this wasn't set to false.

I'll try to fix this, but as a temporary workaround you can call `StepperLayout#setTabNavigationEnabled(false)` after calling `StepperLayout#hideProgress()`

**Comment by zawadz88** [comment]
This should be fixed in `3.2.1` now.
Thanks for reporting this!
